docs/Installation.md

```diff
@@ -73,6 +73,13 @@ Install dependencies not available from binary packages:
 sudo cpanm Daemon::Control JSON::Validator Log::Any Log::Any::Adapter::Dispatch Starman
 ```
 
+Install optional dependencies:

```

Comment:
I think this should be moved to a separate section at the end of the installation instruction where it could be explained what it is about. The normal installation instruction should be as simple as possible.

Reply:
Done